CMSgov/onemac
# OY2 1090 error msg

Story: https://qmacbis.atlassian.net/browse/OY2-1090
Endpoint: https://d36w78p5xnenvn.cloudfront.net

Changes:
- Changed Backend responses so SUCCESS and ERRORS use the same system - switched from string messages to Response Code format.
- Removed the buildAppropriateResponse function -- I had created it in the refactor over the holiday and upon further review, discovered that the Lambdas are wrapped in a function that does the same thing, so no need to do twice.
- pulled up all changeRequestType specific logic into the respective changeRequest files - this created extra fields like the regex and also made several of the form-utils functions unnecessary
- because the functionality left in form-utils was now only used by SubmissionForm, pulled that functionality into SubmissionForm
- Created error-mappings file to convert back end codes into front end alerts
- Updated the s3Uploader to return more appropriate messaging for the issue discovered by ETag header issue
- passed the error up into the SubmissionForm so that it could be displayed to user
- added the contact help desk alert.

Tests:
1. ON THIS BRANCH, I have modified the s3 bucket to contain the error condition about 6MB files.
2. login and verify that trying to upload a 6MB file results in a help desk error
3. verify that smaller files are accepted.

Endpoint: https://d36w78p5xnenvn.cloudfront.net
